app/models/gemini_client.py

```
@@ -93,16 +96,16 @@ def ask(self, prompt: str):
             return response_dict
 
         except Exception as e:
-            print(f"GeminiService ask API呼び出し失敗: {e}", flush=True)
+            logger.exception(f"[ask] GeminiService ask API call failed: {e}")
```

Ruff (0.13.3)

Redundant exception object included in `logging.exception` call

(TRY401)